Plugins/DeleteUnusedBranches/DeleteUnusedBranchesForm.cs

```diff
@@ -162,19 +177,29 @@ private void Delete_Click(object sender, EventArgs e)
             ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
             {
                 await TaskScheduler.Default.SwitchTo(alwaysYield: true);
-
+                GitArgumentBuilder args;
                 if (remoteBranches.Count > 0)
                 {
                     // TODO: use GitCommandHelpers.PushMultipleCmd after moving this window to GE (see FormPush as example)
                     var remoteBranchNameOffset = remoteBranchPrefix.Length;
-                    var remoteBranchNames = string.Join(" ", remoteBranches.Select(branch => ":" + branch.Name.Substring(remoteBranchNameOffset)));
-                    _gitCommands.RunGitCmd($"push {remoteName} {remoteBranchNames}");
+
+                    args = new GitArgumentBuilder("push")
+                    {
+                        remoteName,
+                        remoteBranches.Select(branch => $":{branch.Name.Substring(remoteBranchNameOffset)}")
+                };
```

Comment:
mis-indented

Reply:
![image](https://user-images.githubusercontent.com/3629489/46256829-43f73d00-c47e-11e8-8d7b-4a456db9ca2b.png)

Reply:
This looks untidy and inconsistent. 
Why are you expecting someone to fix it for you in subsequent PRs?

Reply:
I'm not.  It is a WIP pull request.  I didn't see the indent issue. I don't mind the reviews and think they are helpful.  I am not focused on code formatting at this point and time.  Right now I am focused on functionality.